### PR TITLE
Automatically waive fees of COVID collectives

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

File: server/models/Collective.js [hunk 1]
```diff
@@ -681,8 +681,11 @@ export default function (Sequelize, DataTypes) {
           }
 
           // Set default platformFeePercent
-          if (!instance.platformFeePercent) {
-            if (instance.type === 'USER' || instance.type === 'ORGANIZATION') {
+          if (instance.platformFeePercent === null || instance.platformFeePercent === undefined) {
+            // Automatically waive fees for collectives created under the COVID-19 category
+            if (instance.tags?.includes('covid-19')) {
```
Comment (kewitz) [inline server/models/Collective.js]: The fix for the tests is this optional chaining operator.

**Comment by kewitz** [comment]
Sorry about the push and forget. I fixed the issue with the optional chaining operator.

**Comment by znarf** [comment]
@kewitz Since when can we use this chaining operator? Is it from babel?

**Comment by kewitz** [comment]
@znarf yes, it was released on [January 27th](https://github.com/babel/babel/issues/10809#issuecomment-578726419).

**Comment by znarf** [comment]
Ship it!

**Pull request merged**